src/helpers/index.ts

```diff
@@ -83,7 +83,8 @@ export async function binarySearchStorageChange<T>(
 		return await getter(nowApi);
 	};
 
-	while (true) {
+	let cond = true
```

Comment:
what is the rationale that the liner complains about `while (true) {}` and not the other one. 

In other words, I feel like there should be an idiomatic way in TS to write an infinite loop that breaks out internally. Perhaps it is not with `while` 🤔

Reply:
Well, afaik the while should have a changeable condition (e.g. a variable) for eslint not to complain (that's what I did). If u have a 'true' boolean, it understands that this will not exit